cmd/remove.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/rancher/rke/cluster"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+func RemoveCommand() cli.Command {
+	removeFlags := []cli.Flag{
+		cli.StringFlag{
+			Name:   "cluster-file",
```

Comment:
as we imply `cluster` in `rke up/remove`, should we rename this parameter to `config`?

Reply:
It would also align with `generate config` command better